### PR TITLE
Feat: Wishlist On Live

### PR DESCRIPTION
**Pull request by Rishi05051997**
Peer members, please review the cod
wishlist-main
e

### CODE REVIEW

File: Design-System/styles/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    height: 100vh;
+    grid-template-columns:1fr;
+    
+    grid-template-areas: 
+    "header"
+    "wishlist_results"
+    "wishlist_cards"
+    "footer"
+    ;
+}
```
Comment (HarshnaNHaswani) [inline Design-System/styles/wishlist.css]: format this

**Review by HarshnaNHaswani — COMMENTED** [review]
code looks good to me. Can you give a priview. A screenshot/deploy?

File: Design-System/styles/wishlist.css (hunk 1 above)
Comment (Rishi05051997) [reply Design-System/styles/wishlist.css]: Yes I attached SS with this commit

File: Design-System/pages/wishlistPage.html [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+          <ul class="menu left_auto">
+            <!-- <li><a class="text-3" onclick="toggleMenu();">Documentation</a></li>
+                <div class="btn-round">
+                  <button class="btn-primary xl-bdr-radius pad-xs">
+                    <a href="https://www.github.com/">
+                      <span class="iconify" data-icon="akar-icons:github-fill"></span>
+                    </a>
+                  </button>
+                </div> -->
+            <!--- Input with Icon -->
```
Comment (nikhil-sharma-nks) [inline Design-System/pages/wishlistPage.html:78]: if this is not necessary then you should consider removing comments

**Review by nikhil-sharma-nks — COMMENTED** [review]
Everything looks good to be merged! 👍

File: Design-System/pages/wishlistPage.html (hunk 2 above)
Comment (Rishi05051997) [reply Design-System/pages/wishlistPage.html:78]: That will necessary in future mobile responsiveness

**Pull request merged**